### PR TITLE
Add data from exid-types.json to the exid types registry

### PR DESCRIPTION
**Pull request by dthaler**
Copied data from https://github.com/FamilySearch/GEDCOM/blob/main/exid-types.json

In cases where data existed and there was a conflict, copied what was in exid-types.json even though it might be wrong, to make it obvious what the difference was.

### CODE REVIEW

File: uri/exid-types/BillionGraves-CemeteryId.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+%YAML 1.2
+---
+label: BillionGraves Cemetery ID
+
+lang: en
+
+type: uri
+
+uri: https://www.billiongraves.com/cemetery/name/
```
Comment (dthaler) [inline uri/exid-types/BillionGraves-CemeteryId.yaml:9]: I noticed in exid-types.json that some uris end in `/` and some don't.  Shouldn't they be consistent?

The spec says: "If the authority maintains stable URLs for each identifier it issues, it is recommended that the [TYPE](https://gedcom.io/specifications/FamilySearchGEDCOMv7.html#TYPE) payload be selected such that appending the [EXID](https://gedcom.io/specifications/FamilySearchGEDCOMv7.html#EXID) payload to it yields that URL."

If that is taken literally, then for everything that follows this recommendation, shouldn't the uri end in `/` like this one does?  I.e., is the URI for FamilySearch Memory ID, etc. wrong?

File: uri/exid-types/GOV-ID.yaml [hunk 2]
```diff
@@ -1,13 +1,15 @@
 %YAML 1.2
 ---
+label: GOV Historic Gazeteer ID
+
 lang: en
 
 type: uri
 
-uri: https://gedcom.io/exid-type/GOV-ID
+uri: http://gov.genealogy.net/
```
Comment (dthaler) [inline uri/exid-types/GOV-ID.yaml:9]: Here I noticed that exid-types.json (from https://github.com/FamilySearch/GEDCOM/pull/566) had one URI and GOV-ID.yaml (from https://github.com/FamilySearch/GEDCOM-registries/pull/96) had a different URI.  I think the exid-types.json is the correct one, but not sure.

File: uri/exid-types/GOV-ID.yaml [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
 
 specification:
-  - GOV Place ID
+  - Historic Gazeteer place ID
```
Comment (dthaler) [inline uri/exid-types/GOV-ID.yaml:12]: Again here the two files had different strings.  The updated text shows the exid-types.json version, though the lower casing of "place" looks like an inconsistency to me.

File: uri/exid-types/RFN.yaml [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
+
 lang: en
 
 type: uri
 
 uri: https://gedcom.io/terms/v7/RFN
 
 specification:
-  - Record File Number
+  - Permanent Record ID
```
Comment (dthaler) [inline uri/exid-types/RFN.yaml:12]: Here the two files had different values.  Perhaps "Record File Number" should be moved to the label?

File: uri/exid-types/RIN.yaml [hunk 5]
```diff
@@ -1,13 +1,15 @@
...
+
 lang: en
 
 type: uri
 
 uri: https://gedcom.io/terms/v7/RIN
 
 specification:
-  - Record ID Number
+  - Automated Record ID
```
Comment (dthaler) [inline uri/exid-types/RIN.yaml:12]: Here the two files had different values.  Perhaps "Record ID Number" should be moved to the label?

**Comment by albertemmerich** [comment]
I have some comments to Dave's observations:

- in GEDCOM-L we discussed the problem of missíng "/" at the end of the payload in TYPE, too. We think this "/" should be added in all cases where it is missing so far as long the paylod will give a valid link as soon as you add the EXID payload. This is the only way to fulfill the recommendation of the standard to have a valid link - without the / in between the two parts the link will fail.
- please do not take the https://gedcom.io/exid-type/GOV-ID as payload in TYPE for GOV EXID IDs. This part will fail to build a valid link the GOV system object by adding the ID taken from EXID
- the correct payload for TYPE in GOV cases should be: https://gov.genealogy.net/
The "s" in https was missing in my proposal, sorry for that. The link would work, but only because the GOV system automatically forwards the link to the correct https:// one. 
- as for lines 10 resp 12 I would like to propose to have a mixture of both of them and write it as:
"GOV Historic Gazeteer Place ID"
By this we refer to GOV as this is part of the link - and we add the explanation that GOV is a historic gazetteer. GOV is an abbreviation which makes sense in German, but I do not have an English version using G, O, and V :-)

Thank you Dave for your observations!!

If we stay with two versions in exid type registry and in yaml files, please do these modifactions on both sides to have a correct version.

After corrceting and deciding in steering committee I would like to see a possibility of an easy way for users & developpers of applications to find this information. So far most of them would not find the data if they do not get a direct link to it.

File: uri/exid-types/RIN.yaml [hunk 6]
```diff
@@ -38,4 +40,12 @@ specification:
         2 EXID 123abc
         3 TYPE https://gedcom.io/terms/v7/RIN#XYZ
 
+contact: GEDCOM@familysearch.org
+
+change controller: FamilySearch
+
```
Comment (dthaler) [inline uri/exid-types/RIN.yaml:46]:
```suggestion
```

I think RINs are by definition specific to many programs, not specific to FamilySearch, I think the change controller should be the GEDCOM Steering Committee, not FamilySearch per se.

File: uri/exid-types/FindAGrave-CemeteryId.yaml [hunk 7]
```diff
@@ -0,0 +1,20 @@
+%YAML 1.2
+---
+label: 'FindAGrave Cemetery ID'
+
+lang: en
+
+type: uri
+
+uri: https://www.findagrave.com/cemetery
```
Comment (dthaler) [inline uri/exid-types/FindAGrave-CemeteryId.yaml:9]:
```suggestion
uri: https://www.findagrave.com/cemetery/
```

File: uri/exid-types/FindAGrave-MemorialId.yaml [hunk 8]
```diff
@@ -0,0 +1,20 @@
+%YAML 1.2
+---
+label: 'FindAGrave Memorial ID'
+
+lang: en
+
+type: uri
+
+uri: https://www.findagrave.com/memorial
```
Comment (dthaler) [inline uri/exid-types/FindAGrave-MemorialId.yaml:9]:
```suggestion
uri: https://www.findagrave.com/memorial/
```

**Comment by dthaler** [comment]
Approved during GEDCOM Steering Committee meeting

**Pull request merged**